src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double areaCalculator();
```

Comment:
The method name must always be a verb. What to do? **getArea** or **calculateArea**